Script/test-SPM.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e # Any subsequent(*) commands which fail will cause the shell script to exit immediately
+
+PROJECT_NAME=TestProject
+
+# Clean up.
+rm -rf $PROJECT_NAME
+
+mkdir -p $PROJECT_NAME && cd $PROJECT_NAME
+
+# Create the package.
+swift package init
+
+# Create the Package.swift.
+echo "// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+import PackageDescription
+let package = Package(
+    name: \"TestProject\",
+    defaultLocalization: \"en-US\",
+    platforms: [
+        .iOS(.v10)
+    ],
+    products: [
+        .library(
+            name: \"TestProject\",
+            targets: [\"TestProject\"]
+        )
+    ],
+    dependencies: [
+        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", from: "3.0.0-beta.3"),
```

Comment:
we should make core version configurable as well

Reply:
Once core is released I plan to change this to `from: "3.0.0"` that way it always pulls the latest version of core. I'm not sure we should allow us to configure anything other than the latest version of core when running this script. otherwise, if the wrong version of core is used when running the release automation we could get invalid results based on the latest release of core.

Reply:
@nporter-adbe 
Do we really need to add the AEPCore dependency here? https://github.com/adobe/aepsdk-edge-ios/blob/f96a7bf61a3b4bdbf234989139d8cf9a64004606/Package.swift#L31  (AEPEdge -> AEPCore -> AEPRulesEngine, AEPService), I think SPM will find the right AEPCore Package path from above code.